langstream-k8s-deployer/langstream-k8s-deployer-core/src/main/java/ai/langstream/deployer/k8s/agents/AgentResourcesFactory.java

```diff
@@ -223,9 +228,15 @@ public static StatefulSet generateStatefulSet(GenerateStatefulsetParams params)
                         .withTerminationMessagePolicy("FallbackToLogsOnError")
                         .build();
 
-        final ContainerPort port =
+        final ContainerPort portHttp =
                 new ContainerPortBuilder().withName("http").withContainerPort(8080).build();
 
+        final ContainerPort portCustomServiceHttp =
```

Comment:
always ? shouldn't we do it only if the application has a python-service ?

Reply:
we have to tweak the CRD (again) in order to do it.
I am not sure it is worth.

a note:
it is not only for "python-service" but for any agent with componentType=SERVICE.